pymathics/graph/base.py

```diff
@@ -1621,25 +1622,66 @@ def eval_1(self, vertices, edges, evaluation, options):
         )
 
 
-class GraphBox(GraphicsBox):
-    def _graphics_box(self, elements, options):
-        evaluation = options["evaluation"]
-        graph, form = elements
-        primitives = graph._layout(evaluation)
-        graphics = Expression(SymbolGraphics, primitives)
-        graphics_box = Expression(SymbolMakeBoxes, graphics, form).evaluate(evaluation)
-        return graphics_box
+class GraphBox(BoxElementMixin):
```

Comment:
Move boxing out to its own module?

Reply:
I was remiss to mention that `mathics.graph.base` is one of those things like the old `mathics.builtin.lists` which is a mishmash of miscellaneous stuff that hasn't been sorted out.  So we should be _reducing_ its size, not making it larger.

Reply:
Sure. I just try to avoid moving things that I am modifying to make easier to track the changes.